models/intel/machine-translation-nar-en-ru-0001/accuracy-check.yml

```
@@ -0,0 +1,38 @@
+models:
+  - name: machine-translation-nar-en-ru-0001
+    launchers:
+      - framework: dlsdk
+        adapter:
+          type: narnmt
+          vocabulary_file: tokenizer_tgt/vocab.json
+          merges_file: tokenizer_tgt/merges.txt
+          output_name: ArgMax_1407/Squeeze
```

Model has no 'ArgMax_1407/Squeeze' output
```suggestion
          output_name: pred/Squeeze
```

---

@Anna-Mironova omg, how can we determine the output name? We have ONNX with an output node named 'pred'. In my local environment, this matched to ArgMax_1407/Squeeze output name in the IR and accuracy_checker works well with this config.

---

so, I can apply this change, but I can't check it on my side.

---

@Anna-Mironova I checked that the automatically build model has output name pred/Squeeze, and input.yml was fixed.